### PR TITLE
Package node files ci

### PR DESCRIPTION
**Pull request by yhaspel**
In this commit:

* fetch sm node binaries from google cloud bucket (script triggered via travis)
* run package script for all supported OSs (travis)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,11 +9,6 @@ matrix:
         - ELECTRON_CACHE=$HOME/.cache/electron
         - ELECTRON_BUILDER_CACHE=$HOME/.cache/electron-builder
 
-    - os: linux
```
Comment (IlyaVi) [inline .travis.yml]: don't remove linux build, as we talked - make packing unique for mac os

Comment (yhaspel) [reply .travis.yml]: What do you mean by that? Not allow packing for other OS? Some travis configuration?

Comment (IlyaVi) [reply .travis.yml]: Yes, only Mac OS should do the actual packing, Linux should run build and lint

Comment (yhaspel) [reply .travis.yml]: OK

**Pull request merged**